docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,96 @@
+---
+id: meshstack.payment-methods
+title: Payment Methods
+---
+
+# Payment Methods
+
+## Introduction
+
+Just like you pay for a new t-shirt online via your credit card, enterprises also have to be responsible for the costs that they make in cloud platforms. Payment methods are used for allocating cloud costs, all the way back to the responsible department of your organization, for a clear and transparent accounting process.  As the cloud can be mighty with its enormous offering of practically unlimited services, it is important that all meshProjects and meshCustomers are correctly charged and that this information makes its way back to the required systems. Payment methods therefor are the link between the cloud usage and the resulting cost in the context of a meshProject.
+
+There are two types of payment methods:
+
+1) A cost center payment method (indicated via `COST_CENTER`), meaning the costs made by the meshTenants in your meshProjects will be fully charged to the cost center of your organization.
+
+2) A budget payment method (indicated via `COST_LIMITATION`), meaning costs made by the meshTenants in your meshProjects will be charged based on the monetary budget amount that is set on this payment method.
+
+![Payment Methods Lifecycle](assets/payment_methods/payment_method_overall_lifecycle.png)
+
+## The Payment Method Lifecycle: Creation
```

Comment:
We should mention what a substitute payment method is in the lifecycle section. The section mentions it but providing a use case will support understanding

Reply:
I added a use case in the first sentence where we mention the substitute.